### ISSUE TITLE
Enforce single-quotes in Rubocop

### ISSUE THREAD

**Issue by dunkOnIT** [thewca/worldcubeassociation.org#8419]
If anyone else wants to pick this up, it requires the following in `.rubocop.yml`:
```yml
Style/StringLiterals:
  Enabled: true
```

And then running `rubocop -a` to deal with any instances where that rule hasn't been enforced.

I'll try do it later today/on Monday

**Comment by misaka** [comment]
PR created as above, it seems like there isn't enough appetite for this, at the moment at least. Happy to contribute further if this changes.

**Comment by dunkOnIT** [comment]
Yeah unfortunately the risk/reward of trying to go about these changes it too high for the team's capacity at the moment. 

If you're interested in contributing on other issues, let me know and I'd be happy to point out some other good first issues!

**Comment by dunkOnIT** [comment]
We are likely too far down the double quotes road to consider this anytime soon
